src/test/resources/Features/RegisterPage.feature

```diff
@@ -1,6 +1,6 @@
-#Author: your.email@your.domain.com
 Feature: This features verify the Register page for all negative and positive scripts
-Scenario: Check the 
-Given The chrome browser is launched
-And  The Url of app is also launched
-Then verify title of the pages 
```

Comment:
you may also verify title also , so that after clicking button it is redirecting to the same page as expected